docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc

```diff
@@ -1,12 +1,16 @@
 [id="troubleshooting-expired-certificates_{context}"]
 = Troubleshooting expired certificates
 
-Each released `{bin}` binary includes an embedded system bundle that expires 30 days after the release due to certificates embedded in the OpenShift cluster.
+Prior to {rh-prod} 1.2.0, each released `{bin}` binary included a system bundle that expires 30 days after the release due to certificates embedded in the OpenShift cluster.
```

Comment:
Don't we still include the embedded system bundle?

Reply:
'embedded' is repeated twice in the same sentence, which is not so nice. I don't think the 'embedded bundle' VS 'bundle' is really important here.

Reply:
Sure, that makes sense. But the inclusion of "prior to {rh-prod} 1.2.0" implies that since 1.2.0, there is no included system bundle. Does that make sense?

Reply:
Oooh ok, I thought you were ascking about the removal of 'embedded'
Rephrase it to "Prior to {rh-prod} 1.2.0, the system bundle included in each released `{bin}` binary expired 30 days after the release due to certificates embedded in the OpenShift cluster." ?

Reply:
Sounds good to me. :+1: